client/data/block-editor/use-block-editor-nux-status-query.ts

```diff
@@ -9,6 +12,8 @@ export const useBlockEditorNuxStatusQuery = (
 	siteId: string
 ): UseQueryResult< BlockEditorNuxStatus > => {
 	const queryKey = [ 'blockEditorNuxStatus', siteId ];
+	const siteIsAtomic = useSelector( ( state ) => isAtomicSite( state, siteId ) );
+	const siteIsJetpack = useSelector( ( state ) => isJetpackSite( state, siteId ) );
```

Comment:
The `isJetpackSite` check should be enough because every Atomic site is Jetpack as well?

Reply:
Technically yes, but I'd like to keep it explicit and future-proof if you don't mind 😉

Reply:
I don't think that everybody knows that an Atomic site is Jetpack as well -- but this redundant condition is more likely to confuse folks rather than teach them. "If Marin writes a condition like this, then surely there must be sites that are Atomic and not Jetpack?"

Reply:
Haha, fair enough. Let me clean it up 👍